templates/zerver/for/working-groups-and-communities.md

```diff
@@ -0,0 +1,37 @@
+Working groups and communities need different things out of chat than
+companies do. At a company, users are on their company's chat all day. In a
+part-time working group or community, users might check the chat less than
+once a week. In such a group, a good chat system can be the difference
+between robust community and non-existence.
+
+Some of Zulip's earliest users were standards bodies and alumni groups, so
```

Comment:
Maybe put in our RC quote in between the sections here?  We could also ask FHIR whether they'd be up for doing a quote.